quickstart.md

```diff
@@ -5,14 +5,20 @@
 1. You'll need a **Google Cloud project** to authenticate (but not necessarily host) your API requests. Go to the [Google Cloud console](https://console.developers.google.com/) and either use an existing project, or follow these steps to create a new one:
 
     1. Sign in with your Google account, if necessary.
-    1. Click Create, or click the "Project" drop-down at the top of the page and then click _New Project_.
+    1. Click Create, or click the "Project" drop-down at the top of the page and then click **_New Project_**.
     1. Name the project.
     1. Click _Create_ again.
     
    The project is now included in the "Project" drop-down at the top of the page.
 
-1. **Enable the API.** Go to the [Perspective API's overview page](https://console.developers.google.com/apis/api/commentanalyzer.googleapis.com/overview) and click **_Enable_**.
-    
+1. **Enable the API.** 
+    1. Command line:
+       `/google/data/ro/teams/cloud-sdk/gcloud services enable commentanalyzer.googleapis.com`
```

Comment:
Note that this is our _public_ documentation. For people outside Google, the `gcloud` path won't exist. It seems like this is available in external `gcloud` as well though (https://cloud.google.com/sdk/gcloud/reference/services/enable), so I'd suggest removing the `/google/...` prefix and just having this command be `gcloud services enable commentanalyzer.googleapis.com`

Reply:
Thanks, I've made and commited the suggested change.